cli/tools/task.rs

```diff
@@ -721,7 +721,7 @@ fn print_available_tasks(
     )?;
     if let Some(description) = &desc.task.description {
       let slash_slash = colors::italic_gray("//");
-      for line in description.split('\n') {
+      for line in description.lines() {
```

Comment:
This strips out the last line, but I think that's fine?

Reply:
Yeah.